2996_reflection/reflection.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: "Reflection for C++26"
-document: P2996R12
+document: D2996R13
 date: today
-audience: CWG, EWG, LWG
+audience: CWG, LWG
```

Comment:
I also keep forgetting to change the audience on isocpp.org/papers. Maybe this next revision I'll remember. Maybe.